i18n/en.json

```diff
@@ -6548,6 +6548,38 @@
     "translation": "Unable to load mattermost configuration file:  Adding DefaultClientLocale to AvailableLocales."
   },
   {
+    "id": "utils.config.bucket_empty.app_error",
+    "translation": "Failed: S3 bucket empty."
+  },
+  {
+    "id": "utils.config.error_checking_bucket_exist.app_error",
+    "translation": "Error checking if bucket exist."
+  },
+  {
+    "id": "utils.config.error_creating_bucket.app_error",
+    "translation": "Error creating S3 bucket."
```

Comment:
When does this error come up?

Reply:
Not sure but might occur for some permission requirement to S3 or minio storage, or other reasons that failed to create a bucket.